list.go

```diff
@@ -10,12 +10,25 @@ import (
 	"text/tabwriter"
 	"time"
 
+	"encoding/json"
+
 	"github.com/Sirupsen/logrus"
 	"github.com/codegangsta/cli"
 	"github.com/opencontainers/runc/libcontainer"
 )
 
-const formatOptions = `table, json, yaml, xml, or "go=<go-template text>"`
+const formatOptions = `table or json`
+
+type containerList struct {
+	ContainerStates []*containerState `json:"containerlist"`
+}
+
+// containerState represents the platform agnostic pieces relating to a
+// running container's status and state
+type containerState struct {
+	Status string `json:"status"`
+	libcontainer.BaseState
```

Comment:
Ok.  Do you consider the config.json "special"?  I'll map the BaseState fields to new fields to decouple.. but then we get to the config.json fields.  Since those are specification release based and (semi?) public, I'd like to just leave that as is.  Thoughts?